organization/_posts/2020/2020-04-23-coordination.md

```diff
@@ -0,0 +1,210 @@
+ * Refactor the open source project's existing documentation to provide an improved user experience or a more accessible information architecture
+ * Write a conceptual overview of, or introduction to, a product or feature
+ * Create a tutorial for a high-profile use case
+ * Create a set of focused how-to guides for specific task
+ * Create a contributor’s guide
+
+Projects can be standard (3 months) or long running (6 months) starting mid-September. The organization application deadline is however May 4, when Google expects us to have already a good idea (with proposals) of the projects we support.
+
+## Alpaka Training
+
+Event initially foreseen for 27 April had to be postponed. The new dates and a tentative agenda for the training are:
+
+**Mon 29 June - Fri 3 July 2020, 9:00 - 10:30 CEST <https://indico.cern.ch/e/alpaka-tutorial>**
+
+Planning for a virtual meeting with short presentations and breakout training sessions every day. Registrants of the previous event will be contacted and can be moved automatically to the new event. 
+
+NB: No decision yet on how/when to organise the Alpaka hackathon.
+
+## LHCC Review of HL-LHC Software and Computing
+
+*Reminder*: LHCC will review software and computing plans for the HL-LHC, Amber Boehnlein (JLab) will chair. Five 20 page documents provided as input to the committee, from ATLAS, CMS, WLCG, DOMA and from *Common Tools and Community Software* (the HSF takes care of the later)
+
+There will be a relatively short virtual meeting in the week of 18 May, with follow-up reviews later.
+
+[Current HSF Document](https://docs.google.com/document/d/1ai7m7kFyiGgl2tKralJKyPX6rlD9tffU7B6wPj_vpZU/edit?usp=sharing). We had the open review and discussion [last week](https://indico.cern.ch/event/904412/), with additional points gathered in [this Google Doc](https://docs.google.com/document/d/1tnUjW7abhxZyP5lozbIT2GDCztmoU9UzlOtlk4HIFBg/edit?usp=sharing)
+
+### Timeline
+
+- Conclude the Google Docs phase today (Thursday 23)
+- Move to LaTeX tomorrow (Friday 24)
+    - Graeme will announce when that is done
+    - Move into Overleaf first
+    - Then, next week, migrate to the HSF docs repository
+- BibTeX references at this point
+- Final week to make any last changes and address points of coherence
+
+**Current list of open points for each section:**
+
+### General
+
+- We are a bit long, currently:
+    - Intro 2
+    - Generators 5
+    - Simulation 7
+    - Reconstruction 6
+    - Analysis 6
+    - Conclusions 2
+    - **Total 29**
+- LaTeX should allow a squeeze (technical "solution"), but we should look out for opporunities to be more concise
+
+### Introduction
+
+- Reminder that we align expected hardware gains with WLCG/ATLAS/CMS
+    - This was checked and is estimated at ~10% per year as the common figure to be used (the experients also give a 20% pa figure, but that seems over-optimistic)
```

Comment:
the _experients_ also give a 20% _pa_ figure => the **experiments** also give a 20% **per year** figure?

Reply:
*pa* is *per annum*, so it's correct, but maybe not so clear as if spelled out